.circleci/config.yml

```diff
@@ -150,6 +150,10 @@ workflows:
       - run_build_and_test:
           name: test_latest
           latest: true
+## GHC 9 is not supported on stackage yet it seems
```

Comment:
can you make an issue to uncomment this when stackage releases a snapshot with GHC 9, and then link the issue here?

Reply:
Will do!